news/2020-03-13-featured-artist-beatmap-updates-from-the-mappers-guild.md

```diff
@@ -0,0 +1,439 @@
+---
+layout: post
+title: Featured Artist Beatmap Updates from the Mappers' Guild
+date: 2020-03-13 00:00:00 +0000
+---
+
+There's a ton of new beatmaps, completed quests, and medals from the Mappers' Guild celebrating osu!'s featured artists!
+
+![](/wiki/shared/news/banners/mappersguild.png)
+
+Since the [previous update](https://osu.ppy.sh/home/news/2019-11-29-featured-artist-beatmap-updates-from-the-mappers-guild) a lot has been happening in the Guild, with over **20** quests completed, over **100** beatmaps ranked, and **16** new medals for you to earn, all tied to the rapidly growing Featured Artist library!
+
+For those who don't know yet, the Mappers' Guild is a way for mappers with three or more ranked beatmaps to earn rewards for creating beatmaps of songs from the Featured Artist library. If you're interested in the Guild, be sure to look at the [Mappers' Guild website](https://mappersguild.com/) for more information.
```

Comment:
better to link to the mappers guild wiki page. it'll be added in #3280

Reply:
done did